docs/_includes/footer.html

```diff
@@ -1,21 +1,21 @@
-  <div class="footer">
-    <div class="footer-links">
-      {% include links.html %}
-    </div>
-    <div class="copyright">
-      Copyright (c) 2016, Docker Inc.
-    </div>
-  </div>
-  <script src="/style/js/jquery-2.1.4.min.js"></script>
-  <script src="/style/js/highlight.min.js"></script>
-  <script src="/style/js/main.js"></script>
-  <script>
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-    ga('create', 'UA-71407002-1', 'auto');
-    ga('send', 'pageview');
-  </script>  
-  </body>
-</html>
+  <div class="footer">
+    <div class="footer-links">
+      {% include links.html %}
+    </div>
+    <div class="copyright">
+      Copyright (c) 2016-2017 the containerd authors
+    </div>
+  </div>
+  <script src="/style/js/jquery-2.1.4.min.js"></script>
+  <script src="/style/js/highlight.min.js"></script>
+  <script src="/style/js/main.js"></script>
+  <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+    ga('create', 'UA-71407002-1', 'auto');
+    ga('send', 'pageview');
+  </script>  
```

Comment:
The whitespace validation is complaining about trailing whitespace after this line

```
docs/_includes/footer.html:19: trailing whitespace.
+  </script>  
```